packages/cashier/src/stores/withdraw-store.ts

```diff
@@ -270,15 +274,16 @@ export default class WithdrawStore {
 
     @action.bound
     validateWithdrawFromAmount() {
-        let error_message = '';
+        let error_message: string | ReactElement = '';
 
         const { client, modules } = this.root_store;
-        const { balance, currency } = client;
+        const { balance = 0, currency } = client;
```

Comment:
A question: Why do you do this? `balance = 0`

Reply:
can be removed, had left out to remove it